port/win/env_win.cc

```diff
@@ -61,7 +62,8 @@ typedef std::unique_ptr<void, decltype(FindCloseFunc)> UniqueFindClosePtr;
 
 void WinthreadCall(const char* label, std::error_code result) {
   if (0 != result.value()) {
-    fprintf(stderr, "pthread %s: %s\n", label, strerror(result.value()));
```

Comment:
Is there a reason to go through errnoStr here rather than calling the Windows function directly?  Just curious if the redirection is worth it and does not obfuscate the code a bit.

Reply:
yeah, either way works, but as the new function also needs to allocate a buffer, it might be simpler just calling a centralized function errnoStr().